config/src/main/java/org/axonframework/config/EventProcessingModule.java

```diff
@@ -677,6 +681,14 @@ public EventProcessingConfigurer usingSubscribingEventProcessors() {
         return this;
     }
 
+    @Override
+    public EventProcessingConfigurer usingSubscribingEventProcessors(
```

Comment:
Sorry, I see I didn't clearly state my mind here, my apologies.
I thinks this method should do something similar as the `usingSubscribingEventProcessors()`. Hence, it should replace the `defaultEventProcessorBuilder` instance entirely.

I think we could get it to work by doing this:
```java
        this.defaultEventProcessorBuilder = (name, conf, eventHandlerInvoker) ->
                subscribingEventProcessor(name, eventHandlerInvoker, builder.build(name).create(conf));
        return this;
```

In doing so, I would expect that the `private defaultEventProcessor` method does not have any changes at all. Just because that wouldn't be needed anymore I think. So, what do you think, @rsobies?

Reply:
Good idea.